src/server/about/controller.js

```diff
@@ -0,0 +1,19 @@
+const aboutController = {
+  handler: (request, h) => {
+    return h.view('about/index', {
+      pageTitle: 'About',
+      heading: 'About',
+      breadcrumbs: [
+        {
+          text: 'Home',
+          href: '/'
+        },
+        {
+          text: 'About'
```

Comment:
should we add a href?

Reply:
no the final breadcrumb doesn't have a link